frontend/web/src/app/services/order/order.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import {Headers, Http, Response, ResponseOptions} from '@angular/http';
+import {Observable} from 'rxjs';
+import {Order} from '../../model/order';
+import {HttpClientService} from '../http/http-client.service';
+import {HttpClient} from '../http/http.client';
+
+@Injectable()
+export class OrderService {
+  private endpoint: string = HttpClient.orderUrl;
+
+  constructor(private http: Http, private client: HttpClientService) {}
+
+  getOrder(id: number): Observable<Order> {
+    return this.client.get(this.endpoint + '/' + id)
+      .map((data) => {
+        return Order.create(data);
+      });
+  }
+
+  getOrdersByDigitalPartID(digitalPartID): Observable<Order[]> {
+    return this.client.get(this.endpoint + '?digitalPartID=' + digitalPartID)
+      .map((data) => {
+        const orders: Order[] = [];
+        for (let i = 0; i < data.length; i++) {
+          orders[i] = Order.create(data[i]);
+        }
+        return orders;
+
+      });
+  }
+
+  getOrders(): Observable<Order[]> {
+    return this.client.get(this.endpoint)
+      .map((data) => {
+        const orders: Order[] = [];
+        for (let i = 0; i < data.length; i++) {
+          orders[i] = Order.create(data[i]);
+        }
+        return orders;
+
+      });
+  }
+
+  createOrder(order: Order): Observable<Order> {
+    return this.client.post(this.endpoint, order)
+      .map((data) => {
+          return data;
+      });
+  }
+  updateOrder(order: Order): Observable<Order> {
+    return this.client.put(this.endpoint + '/' + order.id, JSON.stringify(order))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)